defaults/main.yml

```diff
@@ -5,3 +5,129 @@ package_state: "present"
 # default vars for the monitoring agent
 monitoring_package_name: stackdriver-agent
 monitoring_service_name: stackdriver-agent
+
+collectd_binary: /opt/stackdriver/collectd/sbin/stackdriver-collectd
+collectd_config_dir: /opt/stackdriver/collectd/etc/collectd.d/managed
+
+# If you define a new plugin template, please add it here. Although a list of
+# just the names of the enabled plugins would be nicer to work with, we manage
+# with a list in which disabled plugins evaluate to empty strings.
+enabled_plugins:
+  - "{% if apache_enabled %}apache{% endif %}"
+  - "{% if cassandra-22_enabled %}cassandra-22{% endif %}"
+  - "{% if cassandra_enabled %}cassandra{% endif %}"
+  - "{% if hbase-098_enabled %}hbase-098{% endif %}"
+  - "{% if hbase-098-standalone_enabled %}hbase-098-standalone{% endif %}"
+  - "{% if hbase-095_enabled %}hbase-095{% endif %}"
+  - "{% if jvm-sun-hotspot_enabled %}jvm-sun-hotspot{% endif %}"
+  - "{% if kafka-082_enabled %}kafka-082{% endif %}"
+  - "{% if memcached_enabled %}memcached{% endif %}"
+  - "{% if mysql_enabled %}mysql{% endif %}"
+  - "{% if nginx_enabled %}nginx{% endif %}"
+  - "{% if redis_enabled %}redis{% endif %}"
+  - "{% if statsd_enabled %}redis{% endif %}"
+  - "{% if tomcat-7_enabled %}tomcat-7{% endif %}"
+
+remove_unmanaged: no
+
+# default vars for third party plugins
+### apache
+
+apache_enabled: no
+apache_name: localhost
+apache_host: local-stackdriver-agent.stackdriver.com
+apache_port: 80
+apache_status_url: "http://{{ apache_host }}:{{ apache_port }}/server-status?auto"
+
+### cassandra 2.2.x
```

Comment:
@qingling128 
It appears cassandra, hbase, kafka, and tomcat are deprecated. Should we still support them in this role?